Cargo.toml

```diff
@@ -43,6 +43,7 @@ members = [
     "crates/optimism/evm/",
     "crates/optimism/node/",
     "crates/optimism/payload/",
+    "crates/optimism/primitives/",
```

Comment:
let me know if I should remove this and put the type contained in the crate back in `reth-primitives`, wanted to make a move in this direction @mattsse